src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -915,38 +924,37 @@ private void AddMessagesToResults()
         {
             foreach (Result result in _results)
             {
-                Rule rule;
-                if (_ruleDictionary.TryGetValue(result.RuleId, out rule))
-                {
-                    Message message = rule.ShortDescription ?? rule.FullDescription;
-                    Dictionary<string, string> replacements = null;
+                int ruleIndex = _ruleIdToIndexMap[result.RuleId];
```

Comment:
>RuleId [](start = 57, length = 6)

Don't you need a test similar to the old code, where you do `TryGetValue` on `_ruleIdToIndexMap`?

Reply:
don't believe so. i think the previous test was unnecessary, as the fortify report consistently contains rules metadata. will verify.

---
In reply to: [249143447](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249143447) [](ancestors = 249143447)

Reply:
verified this data s.be present, no need for check

---
In reply to: [249147574](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249147574) [](ancestors = 249147574,249143447)